### PR TITLE
Doc: Create user-group.md

### PR DESCRIPTION
**Pull request by parniantaghipour**
I need to refer to my tools and prompt set description.

### CODE REVIEW

File: docs/features/user-group.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+:::note
+To add new tools, users must have access to **Tools**. The same applies to managing **Knowledge** and **Prompts**.
+:::
+:::info
+Users cannot view tools, prompts, or knowledge items added by others. However, admins have full visibility and can access all tools, prompts, and knowledge items, regardless of who added them.
+:::
+## Steps to Manage User Groups
+
+1. **Navigate to the Admin Panel**:
+   - Open the Admin Panel and select the **USers** => **Groups** section.
```
Comment (tjbck) [inline docs/features/user-group.md:44]: typo

File: docs/features/user-group.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 3
+title: "👥 User Groups"
+---
+
+# Open WebUI V4: User Group Feature
```
Comment (tjbck) [inline docs/features/user-group.md:6]: Unnecessary title

File: docs/features/user-group.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Overview
+
+The User Group feature introduced in Open WebUI V4 allows users with admin access to define and manage access levels for different groups of users. This feature enhances flexibility by providing the ability to assign varying permissions and roles to users based on their needs and responsibilities.
```
Comment (tjbck) [inline docs/features/user-group.md:10]: It's not v4

File: docs/features/user-group.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+2. **User**: Standard users with permissions defined by the group settings.
+3. **Admin**: Users with access to all features, models, tools, and other system capabilities.
+:::note
+Admins can view the chat history of users, including those with pending access.
+:::
+### Admin Panel: Groups Section
+
+The **Groups** section in the Admin Panel enables administrators to:
+- **Define Access Levels**: Adjust the default permission levels for normal users.
+- **Create New Roles**: Introduce custom user levels with specific permissions.
```
Comment (tjbck) [inline docs/features/user-group.md:29]: you don't create new roles

File: docs/features/user-group.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Administrators can alter these permissions to match the requirements of their organization or project.
+
+:::note
+To add new tools, users must have access to **Tools**. The same applies to managing **Knowledge** and **Prompts**.
+:::
+:::info
+Users cannot view tools, prompts, or knowledge items added by others. However, admins have full visibility and can access all tools, prompts, and knowledge items, regardless of who added them.
+:::
+## Steps to Manage User Groups
```
Comment (tjbck) [inline docs/features/user-group.md:41]: Do not describe it, show it using visual aids.

File: docs/features/user-group.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 3
+title: "👥 User Groups"
+---
+## Overview
+
+The User Group feature introduced in Open WebUI allows users with admin access to define and manage access levels for different groups of users. This feature enhances flexibility by providing the ability to assign varying permissions and roles to users based on their needs and responsibilities.
+
+Administrators can manage users and groups directly from the **Admin Panel**, which includes two main sections: **Users** and **Groups**.
```
Comment (tjbck) [inline docs/features/user-group.md:9]: `User` Page from Admin Panel

File: docs/features/user-group.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 3
+title: "👥 User Groups"
+---
+## Overview
```
Comment (tjbck) [inline docs/features/user-group.md:5]: Not needed.

File: docs/features/user-group.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+1. **Navigate to the Admin Panel**:
+   - Open the Admin Panel and select the **Users** => **Groups** section.
+2. **Create or Modify Groups**:
+   - To create a new group, click **Creat Group** and specify the group name and description.
+   - To modify an existing group, select it from the list and adjust the settings.
+
+3. **Assign Permissions**:
+   - Define the default permissions for users in the group by navigating to the **Groups** => **Edit User Group** => **Permissions**.
+   - Use the setting to give access to Workspace sections (Model, Tools, Promps, Knowledge), and Chat Permissions(File Upload, Chat Edit, Chat Delete, Temporary Chat).
+   :::tip
```
[the remark this answers is not in the document]
Comment (parniantaghipour) [reply docs/features/user-group.md:53]: Why what?

[the remark this answers is not in the document]
Comment (tjbck) [reply docs/features/user-group.md:53]: `Admins are unable to grant users direct access to Functions.`

File: docs/features/user-group.md [hunk 9]
```diff
@@ -22,26 +19,30 @@ There are three default roles for users within a group:
 :::note
 Admins can view the chat history of users, including those with pending access.
 :::
-### Admin Panel: Groups Section
+### Admin Panel: Groups
```
[the remark this answers is not in the document]
Comment (tjbck) [reply docs/features/user-group.md:22]: Do not close comments that's not resolved.

**Review by tjbck — CHANGES_REQUESTED** [review]

File: docs/features/user-group.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 3
+title: "👥 User Groups"
+---
+The User Group feature introduced in Open WebUI allows users with admin access to define and manage access levels for different groups of users. This feature enhances flexibility by providing the ability to assign varying permissions and roles to users based on their needs and responsibilities.
+
+Administrators can manage users and groups directly from the user page from the **Admin Panel**, which includes two main sections: **Users** and **Groups**.
```
Comment (tjbck) [inline docs/features/user-group.md:7]: `User`

File: docs/features/user-group.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+### Groups 
+
+The **Groups** section in the Admin Panel enables administrators to:
+- **Define Access Levels**: Adjust the default permission levels for normal users.
+- **Create New User Groups**: Introduce custom user levels with specific permissions.
+- **Assign Permissions**: Control access to workspace, tools, models, and other features for each group.
+- **Assign Users to Groups**: Assign users to specific groups.
+
+:::note
+To add new tools, users must have access to **Tools**. The same applies to managing **Knowledge** and **Prompts**.
```
Comment (tjbck) [inline docs/features/user-group.md:29]: `have workspace permission`

File: docs/features/user-group.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+1. **Pending**: Users awaiting approval for their account to be activated.
+2. **User**: Standard users with permissions defined by the group settings.
+3. **Admin**: Users with access to all features, models, tools, and other system capabilities.
+:::note
+Admins can view the chat history of users, including those with pending access.
+:::
+### Groups 
+
+The **Groups** section in the Admin Panel enables administrators to:
+- **Define Access Levels**: Adjust the default permission levels for normal users.
```
Comment (tjbck) [inline docs/features/user-group.md:23]: be more explicit here, members with a `user` role.

File: docs/features/user-group.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+- **Create New User Groups**: Introduce custom user levels with specific permissions.
```
Comment (tjbck) [inline docs/features/user-group.md:24]: user levels?

File: docs/features/user-group.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+- **Assign Permissions**: Control access to workspace, tools, models, and other features for each group.
+- **Assign Users to Groups**: Assign users to specific groups.
+
+:::note
+To add new tools, users must have **workspace permission** and access to **Tools**. The same applies to managing **Knowledge** and **Prompts**.
```
Comment (tjbck) [inline docs/features/user-group.md:29]: explicitly mention this is an example.

File: docs/features/user-group.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+:::
+:::info
+Users cannot view tools, prompts, or knowledge items added by others. However, admins have full visibility and can access all tools, prompts, and knowledge items, regardless of who added them.
```
Comment (tjbck) [inline docs/features/user-group.md:32]: Swap the two sentences and start by saying `Admins ....`

File: docs/features/user-group.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+
+![User Tools Permission Demo](/img/tools-user.png)
+#### Users can only view the tools they have added themselves.
```
Comment (tjbck) [inline docs/features/user-group.md:35]: do not use `####` for captions

File: docs/features/user-group.md [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+- **Assign Permissions**: Control access to workspace, tools, models, and other features for each group.
+- **Assign Users to Groups**: Assign users to specific groups.
+
+:::note
+For example, to add new tools, users must have **workspace permission** and access to **Tools**. The same applies to managing **Knowledge** and **Prompts**. 
+:::
+:::info
+Admins have full visibility and can access all tools, prompts, and knowledge items, regardless of who added them, however, Users cannot view tools, prompts, or knowledge items added by others. 
+
+![User Tools Permission Demo](/img/tools-user.png)
```
Comment (tjbck) [inline docs/features/user-group.md:34]: Unnecessary.

File: docs/features/user-group.md (hunk 5 above)
Comment (tjbck) [reply docs/features/user-group.md:42]: This part hasn't been resolved.

**Comment by tjbck** [comment]
Feel free to reopen.

**Pull request closed without merging**